features/lorawan/plugins/FragBDWrapper.h

```diff
@@ -0,0 +1,91 @@
+/*
+ * PackageLicenseDeclared: Apache-2.0
```

Comment:
Not SPDX-License-Identifier instead as in other files, I Can see just two here using this identifier instead

Reply:
@0xc0170 FragAssembler is based on third party code. But yeah, you are right, FragBDWrapper should have SPDX-License-Identifier. Will fix that. 
For testing: Unit tests will be added to the PR once https://github.com/ARMmbed/mbed-os/pull/9219 gets merged. After #9219, we will do a doc update to this PR plus unit tests.  
For protocol implementation testing, we wrote an Application Server that talks to Public TTN LoRaWAN network . It can be found here https://github.com/hasnainvirk/lorawan-ttn-as-test. 
At the moment there is no real life implementation for these protocols (plug-ins). We are the first.  That's why we had to come up with this testing solution. The AS is written in golang, so I am not sure how we can integrate it in CI. However, we would definitely like to see some tests around this.